core/src/main/java/com/microsoft/applicationinsights/internal/profile/CdsRetryPolicy.java

```diff
@@ -0,0 +1,62 @@
+package com.microsoft.applicationinsights.internal.profile;
+
+/**
+ * Responsible for Cds Retry Policy configuration.
+ */
+public enum CdsRetryPolicy {
+    /**
+     * Cached instance to be reused across SDK for CDS Profile fetch calls.
+     */
+    INSTANCE;
```

Comment:
This is a good learning to me, never heard of it before :)

Reply:
Do you mean the use of enum? :)